frontend/src/components/HomePageDesktopFilter/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, {useEffect} from 'react';
+import styled from 'styled-components';
+import {
+    Tab,
+    Tabs,
+    Hidden,
+} from '@material-ui/core'
+
+const DeskTopFilters = styled(Tabs)`
```

Comment:
are you planning to add a style to this? seems a bit odd to have an empty styled component but you do you haha